support-frontend/assets/components/orderSummary/orderSummaryTsAndCs.tsx

```diff
@@ -0,0 +1,126 @@
+`;
+const productStartDate = css`
+	display: block;
+	${textSans14};
+	color: #606060;
+	background-color: ${palette.neutral[97]};
+	border-radius: ${space[3]}px;
+	padding: ${space[3]}px;
+	margin-top: ${space[2]}px;
+	${from.desktop} {
+		margin-top: ${space[4]}px;
+	}
+	li + li {
+		margin-top: ${space[2]}px;
+	}
+`;
+
+interface OrderSummaryStartDateProps {
+	productKey: ActiveProductKey;
+	startDate: string;
+}
+export function OrderSummaryStartDate({
+	productKey,
+	startDate,
+}: OrderSummaryStartDateProps): JSX.Element {
+	const orderSummaryStartDate: Partial<Record<ActiveProductKey, JSX.Element>> =
+		{
+			TierThree: (
+				<ul css={productStartDate}>
+					<li>Your digital benefits will start today.</li>
+					<li>
+						Your Guardian Weekly subscription will start on {startDate}. Please
+						allow 1 to 7 days after your start date for your magazine to arrive,
+						depending on national post services.
+					</li>
+				</ul>
+			),
+		};
+	return orderSummaryStartDate[productKey] ?? <></>;
+}
+
+export interface OrderSummaryTsAndCsProps {
+	productKey: ActiveProductKey;
+	contributionType: ContributionType;
+	countryGroupId: CountryGroupId;
+	promotion?: Promotion;
+	thresholdAmount?: number;
+}
+export function OrderSummaryTsAndCs({
+	productKey,
+	contributionType,
+	countryGroupId,
+	promotion,
+	thresholdAmount = 0,
+}: OrderSummaryTsAndCsProps): JSX.Element {
+	// Proceeds with RegularContributionType only
+	if (contributionType === 'ONE_OFF') {
+		return <></>;
+	}
+	const period = contributionType === 'MONTHLY' ? 'month' : 'year';
+	const tierThreeSupporterPlusTsAndCs = (
+		<div css={containerSummaryTsCs}>
+			{promotion && (
+				<p>
+					You’ll pay{' '}
+					{productLegal(
+						countryGroupId,
+						contributionType,
+						'/',
+						thresholdAmount,
+						promotion,
+					)}{' '}
+					afterwards unless you cancel. Offer only available to new subscribers
+					who do not have an existing subscription with the Guardian.
+				</p>
+			)}
+			{productKey === 'SupporterPlus' && (
+				<>
+					<p>Auto renews every {period} until you cancel.</p>
+					<p>
+						Cancel or change your support anytime. If you cancel within the
+						first 14 days, you will receive a full refund.
+					</p>
+				</>
+			)}
+			{productKey === 'TierThree' && (
+				<p>Auto renews every {period}. Cancel anytime.</p>
+			)}
+		</div>
+	);
+	const contributeAdLiteTsAndCs = (
+		<div css={containerSummaryTsCs}>
+			<p>Auto renews every {period} until you cancel.</p>
+			<p>
+				{productKey === 'GuardianAdLite'
+					? 'Cancel anytime.'
+					: 'Cancel or change your support anytime.'}
+			</p>
+		</div>
+	);
+	const orderSummaryTsAndCs: Partial<Record<ActiveProductKey, JSX.Element>> = {
+		GuardianAdLite: contributeAdLiteTsAndCs,
+		Contribution: contributeAdLiteTsAndCs,
+		SupporterPlus: tierThreeSupporterPlusTsAndCs,
+		TierThree: tierThreeSupporterPlusTsAndCs,
+	};
+	return orderSummaryTsAndCs[productKey] ?? <></>;
```

Comment:
As in the other module, I don't think we need to mulish coalescing operator. 
```suggestion
	return orderSummaryTsAndCs[productKey]
	;
```

Reply:
Not sure how we handle the `undefined` cases though caused by using `Partial`? Think am getting a type error if I do this?

Reply:
@maloky : have applied a `null` check here (ass recommended [below](https://github.com/guardian/support-frontend/pull/6849#discussion_r2007279924))